tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -4874,13 +4874,25 @@ public function propfindResultShouldContainEntries(
 		$elementRows = $expectedFiles->getRows();
 		$should = ($shouldOrNot !== "not");
 		foreach ($elementRows as $expectedFile) {
-			$fileFound = $this->findEntryFromPropfindResponse(
-				$expectedFile[0],
-				$user,
-				$method,
-				"files",
-				$folderpath
-			);
+			$resource = $expectedFile[0];
+			if ($resource === '') {
+				continue;
+			}
+			if ($method === "REPORT") {
+				$fileFound = $this->findEntryFromSearchResponse(
+					$expectedFile[0],
+					$user
+				);
+			} else {
+				$fileFound = $this->findEntryFromPropfindResponse(
+					$expectedFile[0],
+					$user,
```

Comment:
replace ALL `$expectedFile[0]` with `$resource`
```suggestion
				$fileFound = $this->findEntryFromSearchResponse(
					$resource,
					$user
				);
			} else {
				$fileFound = $this->findEntryFromPropfindResponse(
					$resource,
					$user,
```